graph/export.go

```diff
@@ -13,16 +13,15 @@ import (
 	"github.com/docker/docker/registry"
 )
 
-// CmdImageExport exports all images with the given tag. All versions
-// containing the same tag are exported. The resulting output is an
-// uncompressed tar ball.
-// name is the set of tags to export.
-// out is the writer where the images are written to.
+// ImageExportConfig holds list of names to be exported to a output stream.
```

Comment:
Could you complete that one based on the previous comment. ``ImageExportConfig`` is used [there](https://github.com/docker/docker/blob/master/api/server/server.go#L889).